packages/berry-pnpify/sources/index.ts

```diff
@@ -1,125 +1,43 @@
-import fs                   from 'fs';
-import os                   from 'os';
-import path                 from 'path';
+import { patchFs, NodeFS }          from '@berry/fslib';
 
-import { NodePathResolver } from './NodePathResolver';
-import { PnPApiLoader }     from './PnPApiLoader';
-import { PnPApiLocator }    from './PnPApiLocator';
-import { VirtualDirReader } from './VirtualDirReader';
+import fs                           from 'fs';
+import os                           from 'os';
+import path                         from 'path';
 
-function mountVirtualNodeModulesFs() {
+import { NodeModulesFS }            from './NodeModulesFS';
+
+function traceFsCalls() {
```

Comment:
I'd prefer to avoid having debug code within the repository, do you think it would be hard to work on pnpify with adding log as needed rather than with this general tracer?

Reply:
No problem, I'll remove the tracing code from the repo